test/test_ious.py

```diff
@@ -0,0 +1,147 @@
+from typing import List, Callable
+
+import pytest
+import torch
+import torch.fx
+from torch import Tensor
+from torchvision import ops
+
+
+class IouTestBase:
+    @staticmethod
+    def _run_test(target_fn: Callable, test_input: List, dtypes: List[torch.dtype], tolerance: float, expected: List):
+        def assert_close(box: Tensor, expected: Tensor, tolerance):
+            out = target_fn(box, box)
+            torch.testing.assert_close(out, expected, rtol=0.0, check_dtype=False, atol=tolerance)
```

Comment:
Why do you ignore checking the `dtype`? :thinking:

Reply:
See my reasoning below. I think the same will apply for IoU as well.